grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -365,7 +365,7 @@ class StockMovementService {
 
         log.info "Stock movement: ${stockMovement?.shipmentStatusCode}"
 
-        def requisitions = Requisition.createCriteria().list(max: params.maxResults?:10, offset: params.offset?:0) {
+        def requisitions = Requisition.createCriteria().list(max: params.max, offset: params.offset) {
```

Comment:
Why you got rid of fallback options?

Reply:
@awalkowiak If max or offset are null they should not be set to any other value because while downloading stock movements we need all results.